azure-kusto-ingest/azure/kusto/ingest/ingest_client.py

```diff
@@ -66,6 +68,8 @@ def ingest_from_file(self, file_descriptor: Union[FileDescriptor, str], ingestio
         :param azure.kusto.ingest.IngestionProperties ingestion_properties: Ingestion properties.
         """
         file_descriptor = FileDescriptor.get_instance(file_descriptor)
+        ingestion_properties.application_for_tracing = self.application_for_tracing
+        ingestion_properties.user_for_tracing = self.user_for_tracing
```

Comment:
What if the user provided their own values in the ingestion props? it will write over it

Reply:
@yogilad unless i didnt understand correctly, it should be only given from KCSB, and not from user, right? (in which case i do need to remove it as input variables in _init_ im aware)

Reply:
You can overwrite whatever the user wrote there

Reply:
@yogilad @AsafMah Asaf disagrees, should we discuss it for a moment?

Reply:
We discussed this yesterday.
The conclusion was this needs to be overwritten and possibly on the message being queued to the service (blob info or something) and not the ingestion properties themselves.

this also has the benefit of not changing the object the user passed to the API call